web/src/components/core/IssuesAlert.tsx

```diff
@@ -35,7 +38,16 @@ export default function IssuesAlert({ issues }) {
     >
       <List>
         {issues.map((i: Issue, idx: number) => (
-          <ListItem key={idx}>{i.description}</ListItem>
+          <ListItem key={idx}>
+            {i.description}{" "}
+            {i.kind === "solver" ? (
+              <Link to={PATHS.conflicts} variant="link" icon={<Icon name="build_circle" />}>
+                {_("Solve it")}
```

Comment:
BTW, running the code looks to me that "Solve it" looks like an action that will solves the conflict automatically, which is not the case. So, I guess we should go for a different wording.

![Screenshot From 2025-05-19 13-02-04](https://github.com/user-attachments/assets/fb575b57-48fa-4ee3-952f-8302b20a7ee8)

Ideas that comes to my mind:

* Review and solve
* Check and solve
* Manage

Reply:
BTW 2, after using the "isInline" prop for the link, which makes it to get properly underlined, I think the icon is not really needed (nor useful). In fact, it is a PITA to properly align it (I just added few inline styles)

Reply:
I would go for 1. Not having icon is fine for me.

Reply:
Still "solve" word not convincing me. I have opted for "Review and fix" but I'm open to any suggestion you can have. See https://github.com/agama-project/agama/pull/2348/commits/098373a15fd0df8a412705a6de2d904929ed2005

| Before | After |
|-|-|
| ![Screenshot From 2025-05-20 13-32-18](https://github.com/user-attachments/assets/280643de-75ca-4aa7-983f-5cab6c8eeb0a) | ![Screenshot From 2025-05-20 13-32-29](https://github.com/user-attachments/assets/c729740f-5731-4a48-9717-3400b6d0db82) |